__tests__/app.test.js

```diff
@@ -175,6 +175,52 @@ describe("GET /api/articles/:article_id/comments", () => {
       .expect(200)
       .then((response) => {
         expect(response.body.msg).toBe("no comments for this article");
+        expect(response.body.comments).toEqual([]);
+      });
+  });
+});
+describe("POST /api/articles/:article_id/comments", () => {
```

Comment:
I would also like to see tests added for the following:
Status 201, ignores unnecessary properties
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, username does not exist